eval/src/vespa/eval/instruction/mixed_map_function.h

```diff
@@ -7,7 +7,7 @@
 namespace vespalib::eval {
 
 /**
- * Tensor function optimizing map operations on tensors.
+ * Tensor function optimizing in-place map operations on tensors.
```

Comment:
class should ideally be renamed to InplaceMapFunction to match its changed functionality, but I suggest doing that in a separate PR.